giskard/ml_worker/core/savable.py

```diff
@@ -64,7 +64,9 @@ def _load_meta_locally(cls, local_dir, uuid: str) -> Optional[SMT]:
             return None
 
         with open(file, "r") as f:
-            return cls._get_meta_class(**yaml.load(f, Loader=yaml.FullLoader))
+            # PyYAML prohibits the arbitary execution so our class cannot be loaded safely,
+            # see: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation
+            return yaml.load(f, Loader=yaml.UnsafeLoader)
```

Comment:
Shouldn't you add cls._get_meta_class, as before

Reply:
The `UnsafeLoader` allows the arbitrary code execution, which directly constructs an object instead of a dict.